### DISCUSSION TITLE
Running debugger with single-file executable

### DISCUSSION THREAD

**Discussion by masad-frost** [oven-sh/bun#10742] [Q&A]
We're using the [Single-file executable](https://bun.sh/docs/bundler/executables) feature, was wondering if it's possible to include the debugger in the build and run it with the equivalent of `bun --inspect`?

**Comment by lhchavez** [comment]
FWIW I was able to achieve this using bun 1.1.7 by doing the following:

```shell
$ BUN_INSPECT=localhost:6499/prefix ./${myCoolBunBinary} 
--------------------- Bun Inspector ---------------------
Listening:
  ws://localhost:6499/replit
Inspect in browser:
  https://debug.bun.sh/#localhost:6499/prefix
--------------------- Bun Inspector ---------------------
```
